tests/phpunit/CRM/Contribute/Form/ContributionTest.php

```diff
@@ -859,19 +859,26 @@ public function testSubmitWithOutSaleTax() {
   public function testReSubmitSaleTax() {
     $this->enableTaxAndInvoicing();
     $this->relationForFinancialTypeWithFinancialAccount($this->_financialTypeId);
-    $form = new CRM_Contribute_Form_Contribution();
```

Comment:
whoa this diff is hard to read. I extracted a function & added a test - it's not how it looks...